R/volcanoPlot.R

```diff
@@ -77,7 +77,21 @@ volcanoPlot <- function(data, plotly = TRUE, ...){
                                 showarrow = F, xref = 'paper', yref = 'paper',
                                 xanchor = 'right', yanchor = 'bottom', xshift = 0, yshift = 0,
                                 font = list(size = 12, color = "blue")))
-             )
+             ) %>%
+          htmlwidgets::onRender("
+          console.log('hi mom')
```

Comment:
I'm not sure if this is a placeholder or not, but it is funny and doesn't seem to affect the final result. :-)

Reply:
Oh yeah - was trying out a different approach before I figured out `event_data()` thing. Will delete this.